packages/frontend/src/pages/SavedDashboards.tsx

```diff
@@ -94,33 +101,41 @@ const SavedDashboards = () => {
                         />
                     </PageBreadcrumbsWrapper>
 
-                    {userCanManageDashboards && !isDemo && (
-                        <Button
-                            text="Create dashboard"
-                            icon="plus"
-                            loading={isCreatingDashboard}
-                            onClick={() =>
-                                createDashboard({
-                                    name: DEFAULT_DASHBOARD_NAME,
-                                    tiles: [],
-                                })
-                            }
-                            disabled={hasNoSpaces}
-                            title={
-                                hasNoSpaces
-                                    ? 'First you must create a space for this dashboard'
-                                    : ''
-                            }
-                            intent="primary"
-                        />
-                    )}
+                    {userCanManageDashboards &&
+                        !isDemo &&
+                        (dashboards.length > 0 || hasNoSpaces) && (
+                            <Tooltip2
+                                content={
+                                    hasNoSpaces
+                                        ? 'First you must create a space for this dashboard'
+                                        : undefined
+                                }
+                                interactionKind="hover"
+                            >
+                                <Button
+                                    text="Create dashboard"
+                                    icon="plus"
+                                    loading={isCreatingDashboard}
+                                    onClick={handleCreateDashboard}
+                                    disabled={hasNoSpaces}
+                                    intent="primary"
+                                />
+                            </Tooltip2>
+                        )}
                 </PageHeader>
 
                 <ResourceList
                     resourceType="dashboard"
                     resourceIcon="control"
                     resourceList={dashboards}
                     showSpaceColumn
+                    onClickCTA={
+                        isDemo
+                            ? undefined
+                            : hasNoSpaces
+                            ? undefined
+                            : handleCreateDashboard
```

Comment:
I find nested ternary conditions hard to read. Would this work instead ? 
```suggestion
                        isDemo || hasNoSpaces ? undefined : handleCreateDashboard
```

Reply:
And we should check user permission to create dashboard as well